dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/Continuous/FirstOrderFilter.mo

```diff
@@ -0,0 +1,75 @@
+within Dynawo.NonElectrical.Blocks.Continuous;
+
+/*
+* Copyright (c) 2022, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source suite of simulation tools for power systems.
+*/
+
+block FirstOrderFilter "First order filter with absolute and rate limits, and a freezing flag"
```

Comment:
What is the difference with RateLimFirstOrderFreeze ?
Also, could the name reflect more what is included ? It is not a simple FirstOrderFilter, it includes limits, rate lims, freeze

Reply:
RateLimFirstOrderFreeze seems to be indeed a good model for what is required by the IEC norm. The interfaces are optional, the behaviour is implemented in the text view, I think the 1/T factor is not at the same place.

Let's change the name in a further push.